model/riscv_types.sail

```diff
@@ -300,6 +300,15 @@ enum extop_zbb = {SEXTB, SEXTH, ZEXTH}
 
 enum zicondop = {CZERO_EQZ, CZERO_NEZ}
 
+enum wrsop = {WRS_STO, WRS_NTO}
+
+enum WaitReason = {WAIT_WFI, WAIT_WRS_STO, WAIT_WRS_NTO}
+mapping wait_name : WaitReason <-> string = {
+  WAIT_WFI     <-> "WAIT-WFI",
+  WAIT_WRS_STO <-> "WAIT-WRS-STO",
+  WAIT_WRS_NTO <-> "WAIT-WRS-NTO",
+}
+
```

Comment:
Might be worth an

```
overload to_str = {wait_name}
```

It's quite handy for debugging.

Reply:
Good idea; added.